streams-contrib/streams-provider-twitter/src/main/java/org/apache/streams/twitter/converter/util/TwitterActivityUtil.java

```diff
@@ -49,6 +50,7 @@
 import java.util.List;
 import java.util.Map;
 
+import static com.google.common.math.DoubleMath.mean;
```

Comment:
How about using Commons Math package instead? trying to reduce dependency on guava